### DISCUSSION TITLE
Lab Questions Example

### DISCUSSION THREAD

**Discussion by enigma037** [Orchid-DOTNET/BscCSIT-2023#82] [Announcements]
1. LINQ queries - (count, find, list)
2. **CRUD (web app or web API) using entity framework - scaffolding, Razor, dbContext, model class.** Compulsory 
3. SQL Injection prevention example.
4. Runtime Polymorphism using inheritance and method overloading).
5. Exception Handling example( try catch throw finally).
6. Generic Class and method Example.
7. File IO operations example (read the file, find total lines, find the longest words from the file.

**Comment by saroj-17** [comment]
Thank you mam
